.changeset/gentle-elephants-tan.md

```diff
@@ -0,0 +1,9 @@
+---
+'@api3/airnode-deployer': minor
+'@api3/airnode-examples': minor
+'@api3/airnode-node': minor
+'@api3/airnode-validator': minor
+---
+
+Maximal concurrency of serverless functions is set based on the chain settings (maxConcurrency field)
+Add option to disable concurrency reservations for all serverless functions
```

Comment:
Be sure to document this properly, since it is confusing that the `maxConcurrency` is required (and always applies at least for dropping requests), but the disabled check is optional.